Tasks/ANT/package.json

```diff
@@ -1,12 +1,14 @@
 {
-  "name": "Ant",
+  "name": "Agent.Tasks.Ant",
```

Comment:
Don't change the name of the task.  

This is also a convention that is not used anywhere else.  And even if we we're going to do this, it wouldn't be Agent.xxx.  Task are explicitly not part of the agent.